engineering/workflow/default-workflow.md

```diff
@@ -60,29 +60,148 @@ Last day of Iteration
 :::
 
 (meetings:backlog-refinement)=
+
 ### 1. Pre-planning and refinement
 
-The Refinement meetings prepares the Iteration board for the Iteration planning
-meetings, which relies on having sequenced and prioritized tasks to be available in the board's Refined column.
+Tasks in the backlog are continually added and refined asynchronously by the whole team, ready for Iteration planning meetings, which rely on having sequenced and prioritized tasks to be available in the board's Refined column.
```

Comment:
```suggestion
Tasks in the backlog are continually added and refined asynchronously by the whole team, ready for Iteration planning meetings, which rely on having sequenced and prioritized tasks to be available in the board's Up next column.
```

Up next, right? rather than refined?